server/lib/report_server/post_processing/job.ex

```diff
@@ -43,16 +49,43 @@ defmodule ReportServer.PostProcessing.Job do
               # the accumulator is nil on the first line
               header_map = Helpers.get_header_map(row)
               rre_index = header_map["run_remote_endpoint"]
-              # add other index lookups here for future actions
-              {[new_preprocessed_row()], %{rre_index: rre_index}}
+              user_id_index = header_map["user_id"]
+              primary_user_id_index = header_map["primary_user_id"]
+              resource_header_map = Enum.reduce(header_map, %{}, fn {k,v}, acc ->
+                if String.starts_with?(k, "res_") do
```

Comment:
This string matching feels fragile. At least, I guess we should document somewhere that reports can't have any other column that fits this pattern.

Reply:
I've updated it to use a regex to include a digit after the `res_` to make it a little more specific.  I agree it is a little brittle but we control both ends so I think the risk is minimal.